engine/class_modules/monk/sc_monk.cpp

```diff
@@ -1658,6 +1663,7 @@ struct charred_passions_t : base_action_t
       background = dual = proc = true;
       may_crit                 = false;
       base_multiplier          = data().effectN( 1 ).percent();
+      school                   = SCHOOL_FIRE;
```

Comment:
Technically it "worked" previously given the override to mults. It being the wrong school indicates that it wasn't based on the correct spell.